### ISSUE TITLE
Not working on symfony 3

### ISSUE THREAD

**Issue by gawrrell** [Ticketpark/TicketparkBarcodeBundle#1]
I've recently updated from symfony2 to symfony3 and I get this error on cache:clear command:

....vendor\ticketpark\barcode-bundle\Ticketpark\BarcodeBundle\DependencyInjection/../Resou
  rces/config\services.yml" does not contain valid YAML.

  [Symfony\Component\Yaml\Exception\ParseException]
  The reserved indicator "@" cannot start a plain scalar; you need to quote the scalar at line 9 (near "arguments: [@
  ticketpark.file.handler]").

**Comment by gawrrell** [comment]
I think I've found a solution:
I've put all @ references inside double quotes, ex:
`arguments: ["@ticketpark.file.handler"]`

**Comment by sprain** [comment]
Yes, this is necessary in Symfony 3. Would you care creating a pull request with the changes?
